warp/_src/types.py

```
@@ -4332,6 +4371,257 @@ def rebuild(self, constructor: str | None = None):
+        Raises:
+            RuntimeError: If instrumentation is not enabled
+
+        Example:
+            >>> queues = wp.WorkStealingQueues(k=4, m=256, enable_instrumentation=True)
+            >>> # ... launch kernels ...
+            >>> wp.synchronize()
+            >>> if queues.validate_work_assignment():
+            >>>     print("✓ All work processed correctly!")
+        """
+        if not self.has_instrumentation():
+            raise RuntimeError("Instrumentation not enabled, cannot validate work assignment")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)